src/main/webapp/app/overview/header-course.component.html

```diff
@@ -19,6 +19,12 @@ <h6 id="course-header-description" class="fw-normal">{{ courseDescription }}</h6
                         {{ 'artemisApp.courseOverview.manage' | artemisTranslate }}
                     </button>
                 </div>
+
+                <div class="d-flex align-items-center justify-content-center manage-button-container" *ngIf="shouldShowGoToStudentViewButton()">
+                    <button class="btn btn-secondary manage-button" (click)="redirectToStudentView()">
+                        {{ 'artemisApp.courseOverview.studentView' | artemisTranslate }}
+                    </button>
```

Comment:
Please make this a link instead of a button to allow opening it in a new tab (see #7584, #611)

Reply:
Good point. I replaced my new button and also the already existing management button 👍